src/bundle/Resources/public/js/scripts/sidebar/extra.actions.btn.js

```diff
@@ -0,0 +1,9 @@
+(function () {
+    const btns = document.querySelectorAll('.ez-btn--extra-actions');
+
+    btns.forEach(btn => {
```

Comment:
Is forEach on NodeList finally supported by all browser or you are missing `[...btn]`?

Reply:
I've found an interesting approach here: https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/10638731/#comment-5

Do you agree with this proposal? IMO it's a bit risky (always adding/overriding native methods is risky).

Reply:
I also think this kind of hack is risky and would avoid if can. Here we easily can do `[...btns]` so I would stay with it.